posthog/gzip_middleware.py

```diff
@@ -0,0 +1,35 @@
+import re
+from typing import List
+
+from django.conf import settings
+from django.middleware.gzip import GZipMiddleware
+
+
+class InvalidGzipAllowList(Exception):
```

Comment:
Nit: Django uses "GZip" capitalization, so let's keep it consistent 😅 
```suggestion
class InvalidGZipAllowList(Exception):
```